_layouts/layout.html

```diff
@@ -17,6 +17,9 @@
     <link rel="stylesheet" href="/static/css/bootstrap.min.css">
     <link rel="stylesheet" href="/static/css/main.css">
     <link rel="stylesheet" href="/static/css/syntax.css">
+
+    <script src="/static/js/browserDetect.js"></script>
```

Comment:
Should be moved to the end of `index.html` (or maybe a new `browserdetect.html` in `_includes/`, and included near the analytics insertion in `layout.html`), otherwise TypeError(?) may get raised